examples/sai_dma_passthru.rs

```diff
@@ -45,6 +46,22 @@ const PLL3_P_HZ: Hertz = Hertz(AUDIO_SAMPLE_HZ.0 * 257);
 static mut TX_BUFFER: [u32; DMA_BUFFER_LENGTH] = [0; DMA_BUFFER_LENGTH];
 #[link_section = ".sram3"]
 static mut RX_BUFFER: [u32; DMA_BUFFER_LENGTH] = [0; DMA_BUFFER_LENGTH];
+pub const CLOCK_RATE_HZ: Hertz = Hertz(400_000_000_u32);
+
+const HSE_CLOCK_MHZ: MegaHertz = MegaHertz(16);
+
+// PCLKx
+const PCLK_HZ: Hertz = Hertz(CLOCK_RATE_HZ.0 / 4);
+// 49_152_344
```

Comment:
Not sure what this number is doing in the comment here, perhaps remove it to avoid confusion

Reply:
Sure, I can remove those. I put them in for debugging some stuff but they shouldn't be needed.